modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/HashAggregatePlannerTest.java

```diff
@@ -43,25 +47,85 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import static org.apache.calcite.rel.RelFieldCollation.Direction.ASCENDING;
+import static org.apache.calcite.rel.RelFieldCollation.Direction.DESCENDING;
+
 /**
  *
  */
 @SuppressWarnings({"TooBroadScope", "FieldCanBeLocal", "TypeMayBeWeakened"})
 public class HashAggregatePlannerTest extends AbstractAggregatePlannerTest {
+    /** */
+    @Test
+    public void indexMinMax() throws Exception {
+        IgniteSchema publicSchema = new IgniteSchema("PUBLIC");
+
+        RelCollation[] collations = new RelCollation[] {
+            RelCollations.of(new RelFieldCollation(1, ASCENDING)),
```

Comment:
Use `TraitUtils.createFieldCollation` method instead of direct `RelFieldCollation` constructor, default nulls ordering for RelFieldCollation is not the same as Ignite's default.